Makefile

```diff
@@ -647,7 +660,7 @@ build/bin: build/bin/kurl
 
 build/bin/kurl:
 	CGO_ENABLED=0 go build $(LDFLAGS) -o build/bin/kurl $(BUILDFLAGS) ./cmd/kurl
-	ldd build/bin/kurl | grep -q "not a dynamic executable" # confirm that there are no linked libs
+	/bin/sh -c "${LDD_OR_OTOOL} build/bin/kurl" #| grep -q "not a dynamic executable" # confirm that there are no linked libs
```

Comment:
im not sure about this change. this will fail a build if its a dynamic executable which is necessary.

Reply:
oh whoops I didn't realize I had that commented.